src/libraries/AppStorage.sol

```diff
@@ -94,4 +94,6 @@ struct AppStorage {
     address feeToken;
     address stratosphereAddress;
     uint256 reentrancyGuardStatus;
+
+    address emissionsManager;
```

Comment:
we will deploy a new diamond, can you move this into the "general" section

Reply:
it's already in general. I have removed the blank line in betwee.